charts/rancher-istio/1.5.800/templates/istio-onefive-migration.yaml

```diff
@@ -59,7 +59,7 @@ metadata:
   annotations:
     "helm.sh/hook": pre-upgrade
     "helm.sh/hook-weight": "2"
-    "helm.sh/hook-delete-policy": hook-succeeded
```

Comment:
What's the reason for changing this policy?

Reply:
Only in order to debug easier. Removing on success cleans up resources, but this is a non-standard process so I figured it would be best to keep around for logs, shouldn't hurt to much to keep one job around. If you try to edit app it deletes as soon as upgrade starts.